local/php/envs.go

```diff
@@ -37,11 +37,30 @@ func (p *Server) generateEnv(req *http.Request) map[string]string {
 	}
 
 	pathInfo := req.URL.Path
+	filenameMatch := false
+
 	if pos := strings.Index(strings.ToLower(pathInfo), ".php"); pos != -1 {
 		file := pathInfo[:pos+4]
 		if _, err := os.Stat(filepath.Join(p.documentRoot, file)); err == nil {
 			scriptName = file
 			pathInfo = pathInfo[pos+4:]
+			filenameMatch = true
+		}
+
+	}
+
+	if pathInfo != "/" && filenameMatch == false {
```

Comment:
Would this work on Windows as Windows separate paths with `\\`?

Reply:
Ugh, actually, that line is probably not a problem, it's just small optimisation, but the other ones might be an issue. Frankly I would have to set up windows for it and it's not quick thing to do. Will have a think, maybe some friend can test it for me.

Reply:
or maybe use [filepath.Separator](https://pkg.go.dev/path/filepath#pkg-constants) instead of hardcoding `"/"`? :)

Reply:
I added a more proper way of dealing with it. Should be windows proof, but then again, i don't have windows :(